taxi/urls.py

```diff
@@ -44,7 +47,24 @@
     path("cars/<int:pk>/delete/", CarDeleteView.as_view(), name="car-delete"),
     path("drivers/", DriverListView.as_view(), name="driver-list"),
     path(
-        "drivers/<int:pk>/", DriverDetailView.as_view(), name="driver-detail"
+        "drivers/create/",
+        DriverCreateView.as_view(),
+        name="driver-create"
+    ),
+    path(
+        "drivers/<int:pk>/update/",
+        DriverUpdateView.as_view(),
+        name="driver-update"
+    ),
+    path(
+        "drivers/<int:pk>/delete",
```

Comment:
```suggestion
        "drivers/<int:pk>/delete/",
```
Don't forget to add a `/` at the end of endpoints to avoid unexpectable 404 errors.